src/groundStation/groundStation.py

```diff
@@ -62,13 +64,13 @@ def __init__(self, opts):
         if opts.interface == 'zmq':
             self.__zmq__(self.myAddr)
         elif opts.interface == 'uart':
-            self.__uart__(opts.device)
+            self.ser = self.__uart__(opts.device)
         elif opts.interface == 'fifo':
             self.__fifo__()
         libcsp.route_start_task()
         time.sleep(0.2)  # allow router task startup
         self.rdp_timeout = opts.timeout  # 10 seconds
-        libcsp.rdp_set_opt(4, self.rdp_timeout, 1000, 1, 250, 2)
+        libcsp.rdp_set_opt(4, self.rdp_timeout, 2000, 0, 1500, 0)
```

Comment:
What does this do?

Reply:
Changes the RDP rate of requests and acks. It resolved an issue we were seeing where there would be multiple packets sent to the OBC bc of the short timeouts here. (https://albertasat.slack.com/archives/C5MJN5YN6/p1634745615466400)

https://github.com/AlbertaSat/libcsp/blob/d5d5f04d1e3d8b73eb322906a1967ee88f7bbb9a/src/transport/csp_rdp.c#L1116

Reply:
oh, cool